proto/api.proto

```diff
@@ -9,6 +9,16 @@ import "google/api/visibility.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 
 option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  security_definitions: {
+    security: {
+      key: "Authorization";
+      value: {
+        type: TYPE_API_KEY;
+        in: IN_HEADER;
+        name: "Authorization";
+      };
+    }
+  };
```

Comment:
I think there's another `security` field you can include (on same level as `security_definitions`) to apply it global: https://swagger.io/docs/specification/authentication/

something like
```
  security: [
    {
      security_requirement: {
        key: "Authorization";
        value: {
          scope: []
        }
      }
    }
  ]
```
?